components/centraldashboard/app/api_contributors.ts

```diff
@@ -1,28 +1,94 @@
-import {Router, Request, Response, NextFunction} from 'express';
-import {DefaultApi} from './clients/profile_controller';
+import {V1Namespace} from '@kubernetes/client-node';
+import {Router, Request, Response} from 'express';
+import {Binding as WorkgroupBinding, DefaultApi} from './clients/profile_controller';
+
 import {
     apiError,
-    mapWorkgroupBindingToSimpleBinding,
-    mapSimpleBindingToWorkgroupBinding,
     ERRORS,
 } from './api';
 
+// From: https://www.w3resource.com/javascript/form/email-validation.php
+const EMAIL_RGX = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+
 // Valid actions for handling a contributor
 type CONTRIBUTOR_ACTIONS = 'create' | 'remove';
```

Comment:
Type names should follow the normal naming rules and use UpperCamelCase.

http://go/tsstyle#identifiers

Reply:
Gotcha